gui/js/updater.window.js

```diff
@@ -75,11 +78,15 @@ module.exports = class UpdaterWM extends WindowManager {
       log.info({ update: info }, 'No update available')
       this.afterUpToDate()
     })
-    autoUpdater.on('error', err => {
-      // May also happen in dev because of code signature error. Not an issue.
-      log.error({ err }, 'Error in auto-updater! ')
-      this.clearTimeoutIfAny()
-      this.afterUpToDate()
+    autoUpdater.on('error', async err => {
+      if (this.skipped) {
+        return
+      } else if (err.code === 'ENOENT') {
+        this.skipUpdate('assuming development environment')
+      } else {
+        await Promise.delay(UPDATE_RETRY_DELAY)
+        autoUpdater.checkForUpdates()
```

Comment:
Shouldn't we still clear the timeout here?

Reply:
Wouldn't this make for an infinite update check?

Reply:
I was confused. I took this call to `autoUpdate.checkForUpdates()` for one to `this.checkForUpdates()`…
We don't need to clear anything here.